File-Formats/DocIO/Working-with-Comments.md

```diff
@@ -139,6 +139,130 @@ document.Close()
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Comments/Modify-text-of-an-existing-comment).
 
+## Inserting comments
+
+You can insert comments into the Word document using the [AppendComment](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.WParagraph.html#Syncfusion_DocIO_DLS_WParagraph_AppendComment_System_String_) and [AddCommentedItem](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.WComment.html#Syncfusion_DocIO_DLS_WComment_AddCommentedItem_Syncfusion_DocIO_DLS_IParagraphItem_) methods.
+
+The following code example illustrates how to find all occurrences of a text pattern using Regex and insert comments to them.
+
+{% tabs %}
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+using (FileStream fileStream = new FileStream("Input.docx", FileMode.Open, FileAccess.ReadWrite))
+{
+    //Open the existing Word document.
+    using (WordDocument document = new WordDocument(fileStream, FormatType.Docx))
+    {
+        //Find all occurrence of a particular text end with comma in the document using regex.
```

Comment:
//Find all occurrence of a particular text ending with comma in the document using regex.